projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/summary.service.ts

```diff
@@ -21,10 +21,12 @@ export class ScatterDataPointSummaryService implements SummaryService {
   summarize(layer: DataLayer, range: NumberRange): Record<string, string>[] {
     const current = this.stats.getFormattedStatistics(layer, range);
     const prev = this.stats.getFormattedStatistics(layer, previous(range));
+    let timeSpan = current['Days Reported'].split('/');
+    timeSpan = timeSpan[1].split(')')
```

Comment:
This is rather brittle and it's more complex that just computing the number of days again here. It's generally not a good practice to parse values from a formatted string unless there's no other way to get that information.

The code to compute number of days from the range is pretty simple:
```
const days = Math.ceil((range.max - range.min) / MILLISECONDS_PER_DAY)
```

If you don't like duplicating it, you could factor it out into a separate function.

Reply:
okay @semicolin